requirements.txt

```diff
@@ -1,5 +1,5 @@
 bs4==0.0.1
-GitPython==3.1.31
+GitPython==3.1.32
```

Comment:
Have you checked all in-app functionality functionality related to this module upgrade on all platforms?

Reply:
I can only check for windows if you don't want this change you can decline it

Reply:
![image](https://github.com/RimSort/RimSort/assets/102756485/8c4bf55a-3f0d-49da-bd8c-e616456a42e4)

But I have no idea why it says 3.1.31 instead of 3.1.32 (maybe it can't grab the latest) should force it to test it out?